src/main/java/com/booking/superride/controller/CustomerController.java

```diff
@@ -1,26 +1,26 @@
-package com.booking.superride.controller;
-
-import com.booking.superride.domain.CustomerDetailsDTO;
-import com.booking.superride.entity.CustomerDetails;
-import com.booking.superride.service.CustomerService;
-import jakarta.validation.Valid;
-import lombok.RequiredArgsConstructor;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
-
-@RestController
-@RequestMapping("/customer")
-@RequiredArgsConstructor
-public class CustomerController {
-
-    private final CustomerService customerService;
-
-    @PostMapping("/add-customer")
-    public ResponseEntity<CustomerDetails> saveCustomerDetails(@RequestBody @Valid CustomerDetailsDTO customerDetailsDTO){
-        return ResponseEntity.ok(customerService.saveCustomerDetails(customerDetailsDTO));
-    }
-
-}
+package com.booking.superride.controller;
+
+import com.booking.superride.domain.CustomerDetailsDTO;
+import com.booking.superride.entity.CustomerDetails;
+import com.booking.superride.service.CustomerService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/customer")
+@RequiredArgsConstructor
+public class CustomerController {
+
+    private final CustomerService customerService;
+
+    @PostMapping("/add-customer")
```

Comment:
Rename the endpoint, use verbs. @Post itself means add so no need to use add also avoid same name in endpoint twice so better name will be leaving it without assigning any name